.pre-commit-config.yaml

```
@@ -59,6 +59,12 @@ repos:
         files: ^web/.*\.(ts|tsx|js|css|html|json)$
         args:
           ["--config=web/.prettierrc.js", "--ignore-path=web/.prettierignore"]
+  
+  - repo: https://github.com/myint/autoflake
```

```suggestion
  - repo: https://github.com/pycqa/autoflake
```
Looks like they have moved repo